scripts/protoc-swagger-gen.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euox pipefail
+
+cd ./proto
+
+
+# find all proto directories
+proto_dirs=$(find ./cheqd -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
+
+# loop through proto directories
+for dir in $proto_dirs; do
+  # generate swagger files (filter query files)
+  query_file=$(find "${dir}" -maxdepth 1 \( -name 'query.proto' \))
+  if [[ -n "$query_file" ]]; then
+    buf generate --template buf.gen.swagger.yaml "$query_file"
```

Comment:
THIS is the main thing - it's the bit that looks at our own proto directories and generates does buf generate. Check if there's a buf generate plugin for creating Swager files, which might be the easiest way to do it.